### ISSUE TITLE
关于torch_inf.py中没有归一化的问题

### ISSUE THREAD

**Issue by songyuanwc** [Intellindust-AI-Lab/DEIMv2#63]
def process_image(model, device, file_path, size=(640, 640)):
    im_pil = Image.open(file_path).convert('RGB')
    w, h = im_pil.size
    orig_size = torch.tensor([[w, h]]).to(device)

    transforms = T.Compose([
        T.Resize(size),
        T.ToTensor(),
    ])
    im_data = transforms(im_pil).unsqueeze(0).to(device)

    output = model(im_data, orig_size)
    labels, boxes, scores = output

    draw([im_pil], labels, boxes, scores)  这个函数没有用            
T.Normalize(mean=[0.485, 0.456, 0.406], 
std=[0.229, 0.224, 0.225])  归一化啊 ，但是我看训练和测试都有添加

**Comment by capsule2077** [comment]
感谢提醒，这是推理中的一个bug，对于S、M、L、X模型是需要归一化的。
